src/routes/safe/components/Transactions/TxsTable/ExpandedTx/index.jsx

```diff
@@ -55,68 +57,109 @@ const ExpandedTx = ({ cancelTx, tx }: Props) => {
     }
   }
 
+  const RenderIncomingTx = ({ tx }) => {
```

Comment:
why are those components defined inside `ExpandedTx` component? why are they prefixed with `Render`?

Reply:
Should be better now